contracts/src/v0.8/dev/KeeperRegistryDev.sol

```diff
@@ -457,7 +461,11 @@ contract KeeperRegistryDev is
   function getMaxPaymentForGas(uint256 gasLimit) public view returns (uint96 maxPayment) {
     (uint256 gasWei, uint256 linkEth) = _getFeedData();
     uint256 adjustedGasWei = _adjustGasPrice(gasWei, false);
-    return _calculatePaymentAmount(gasLimit, adjustedGasWei, linkEth);
+    bytes memory data = new bytes(0);
+    if (PAYMENT_MODEL == PaymentModel.OPTIMISM) {
+      data = bytes.concat(ESTIMATED_MSG_DATA, L1_FEE_DATA_PADDING);
```

Comment:
`ESTIMATED_MSG_DATA` is a bug confusing, maybe MAX_MSG_DATA or something? 

Also I realise that 32 bytes is too small. See above comment about `PERFORM_DATA_PADDING`, we'll need at least that.

Reply:
Done